easybuild/easyconfigs/s/STAR/STAR-2.3.1z12-goolf-1.7.20.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'STAR'
+version = '2.3.1z12'
+
+homepage = 'https://code.google.com/p/rna-star/'
+description = """STAR: ultrafast universal RNA-seq aligner"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/alexdobin/%(name)s/archive/']
+sources = ['%(name)s_%(version)s.tar.gz']
+
+checksums = ['c1e5b36a02011a73f9461f5bdc4dacaa']
+
+buildopts = 'CXX="$CXX"'
+
+executables = [name, 'STARstatic']
+files_to_copy = [(executables,'bin/')]
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['bin']
```

Comment:
this is a bit silly, since you're already checking for `bin/STAR`, so make the `dirs` bit an empty list? `[]`